src/plugins/intel_gna/tests/unit/transformations/gna_insert_copy_layer.cpp

```diff
@@ -1,21 +1,25 @@
-// Copyright (C) 2022 Intel Corporation
+// Copyright (C) 2023 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 //
 
 #include <gtest/gtest.h>
 
+#include <functional_test_utils/include/functional_test_utils/precision_utils.hpp>
 #include <legacy/ngraph_ops/crop_ie.hpp>
 #include <ngraph/function.hpp>
-#include <ngraph/opsets/opset8.hpp>
 #include <ngraph/pass/manager.hpp>
 #include <transformations/init_node_info.hpp>
 #include <transformations/utils/utils.hpp>
 
 #include "common_test_utils/ngraph_test_utils.hpp"
 #include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/subgraph_builders.hpp"
+#include "openvino/opsets/opset11.hpp"
 #include "ops/copy.hpp"
 #include "transformations/insert_copy_layer.hpp"
 
+using namespace ov::opset11;
+
```

Comment:
```suggestion
using namespace ov;
using namespace ov::element;
```

Reply:
Ok, I will remove all ov:: from this file.